openpectus/aggregator/aggregator.py

```diff
@@ -26,15 +25,89 @@ def __init__(self, engine_data_map: EngineDataMap, publisher: FrontendPublisher)
         self.publisher = publisher
 
     def register_engine_data(self, engine_data: EngineData):
+        # Why do we do this?
```

Comment:
Is this comment, and the commented out code below, relevant to keep, or should it just be deleted?

Reply:
Should be deleted. Is one of the leftovers to be dealt with in #380